person.rb

```diff
@@ -11,6 +11,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] To implement the `has-many` side of the relationship(a person has many rentals), a method for adding rentals should be present, something like so: 

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```
Please kindly fix that.

Reply:
Well solved